src/main/java/com/microsoft/aad/msal4j/AccountCacheEntity.java

```diff
@@ -49,6 +50,8 @@ ClientInfo clientInfo() {
     @JsonProperty("authority_type")
     protected String authorityType;
 
+    protected Map<String, ?> idTokenClaims;
```

Comment:
how this will be populated ?

Reply:
It is populated in the create() method, and the Account objects get it through the toAccount() method. Is there a situation where an AccountCacheEntity isn't created for an account, or a situation where we return an Account object that doesn't come from the AccountCacheEntity?

However, you also say that token claims "shouldn't be part of the key", but if the Account class now has token claims as a field, shouldn't the cache entity match? If not, where would the Account/IdToken relation come from? As far as I can tell, the IdToken object is only set in TokenRequestExecutor.createAuthenticationResultFromOauthHttpResponse(), which is also where AccountCacheEntities are created.